src/SyntaxError.js

```diff
@@ -0,0 +1,24 @@
+import formatCodeFrame from 'babel-code-frame';
+
+export default class SyntaxError extends Error {
+  constructor(error) {
+    super(error);
+
+    const { reason, message, line, column, source } = error;
+
+    this.name = 'SyntaxError';
+    this.message = reason || message;
+
+    if (line && column) {
+      this.message += ` (${line}:${column})`;
+
+      if (source) {
+        this.message += `\n\n${formatCodeFrame(source, line, column)}\n`;
```

Comment:
Please use the correct PostCSS API (`err.showSourceCode()`) instead of `babel-code-frame` here, it's the code frame for CSS with correct syntax highlighting and always defined when uning `err.name === 'CssSyntaxError' ? new SyntaxError(err) : err`

Reply:
@michael-ciniawsky yep, only one interesting, when you have invalid css code `showSourceCode` nothing returns :confused: